ocs_ci/ocs/node.py

```diff
@@ -219,3 +223,47 @@ def get_node_ips(node_type='worker'):
         ]
     else:
         raise NotImplementedError
+
+
+def add_new_node_and_label_it(machineset_name):
+    """
+    Add a new node and label it
+
+    Args:
+        machineset_name (str): Name of the machine set
```

Comment:
can you add an example what will be machineset_name

Reply:
Sure, will add it in my next PR - IPI reactive case.